CHANGELOG.md

```diff
@@ -78,6 +78,11 @@ All notable, unreleased changes to this project will be documented in this file.
 
 ### Breaking changes
 
+- **Feature preview breaking change**:
+  - `orderBulkCreate` now will attempt to create order with `IGNORE_FAILED` policy even if: - #14177 by @kadewu
+    - `User` cannot be resolved, and `email` wasn't provided.
+    - `Variant` wasn't provided.
```

Comment:
```suggestion
  - Order bulk create allow to import archive orders - #14177 by @kadewu
    - `orderBulkCreate` now will attempt to create order with `IGNORE_FAILED` policy even if:
      - `User` cannot be resolved, and `email` wasn't provided.
      - `Variant` wasn't provided.
```

Reply:
I'm also wondering if those changes are really breaking? 🤔 Like we are not cutting the possibility, but extending it

Reply:
Actually, I also wondered how to address that change, I would like to make sure, that clients that are using already that mutations are aware, that they now create some orders when using said policy. But I'm fine with changing where we include that information.

Reply:
@maarcingebala what do you think?